internal/services/postgres/postgresql_flexible_server_virtual_endpoint_resource_test.go

```diff
@@ -0,0 +1,121 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package postgres_test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/go-azure-sdk/resource-manager/postgresql/2023-06-01-preview/virtualendpoints"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/acceptance"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/acceptance/check"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/clients"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/services/postgres"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/pluginsdk"
+	"github.com/hashicorp/terraform-provider-azurerm/utils"
+)
+
+type PostgresqlFlexibleServerVirtualEndpointResource struct{}
+
+func TestAccPostgresqlFlexibleServerVirtualEndpoint_basic(t *testing.T) {
```

Comment:
According to the schema `replica_server_id` is updateable, so it would be good to add a test that validates the update path.

Reply:
A test has been added to cover update operations that change the replica id